examples/example_host.cpp

```diff
@@ -1,12 +1,15 @@
-#include <evmc/evmc.h>
-#include <evmc/helpers.h>
+// EVMC -- Ethereum Client-VM Connector API
+// Copyright 2018 The EVMC Authors.
+// Licensed under the Apache License, Version 2.0. See the LICENSE file.
+
+/// @file
+/// Example implementation of an EVMC Host.
 
-#include <inttypes.h>
-#include <stdio.h>
-#include <stdlib.h>
-#include <string.h>
+#include "example_host.h"
+
+#include <evmc/helpers.h>
 
-evmc_uint256be balance(evmc_context* context, const evmc_address* address)
+static evmc_uint256be balance(evmc_context* context, const evmc_address* address)
```

Comment:
Not relevant to this PR, but I guess if this is `.cpp` we should use an anonymous namespace instead?

Reply:
I'm trying to stay in C dialect as much as possible.